lib/reference.js

```diff
@@ -5,39 +5,39 @@ var Reference = NodeGit.Reference;
 var Branch = NodeGit.Branch;
 
 /**
-* Retrieves the reference pointed to by the oid
+* Retrieves the reference by it's short name
```

Comment:
Fair, but thats not new, it's preexisting (just looks new because of how the diff is shown)